src/ui/component/stepper/stepperProgress/stepperProgress.tsx

```diff
@@ -0,0 +1,73 @@
+import type { FC } from 'react'
+import type { ProgressBarTransition } from './progressBar/progressBar'
+import { ProgressBar } from './progressBar/progressBar'
+import './stepperProgress.scss'
+
+export type StepId = string
+export type StepStatus = 'error' | 'disabled' | 'complete' | 'incomplete'
+
+export type StepData = {
```

Comment:
I will have simplified by only `Step`, that is self-explanatory enough

Reply:
A `Step` type is created for the Step component in the stepper PR, that's why I tried to create an other one for step data. We can change `StepData` to `Step` and probably update the other PR `Step` type to `StepElement` or something similar. What do you think?

Reply:
You can scope the `Step` type to the `stepperProgress` component or create a sub-type inherited from `Step` in `stepper` component, because you handle the same properties here no? (`id`, `order`, `status`)

Reply:
Yes! We can definitely create such relationship using a sub-type. It makes the component easier to read. Thanks!